UIKitDZ/UIKitDZ/ViewController.swift

```diff
@@ -9,9 +9,161 @@ import UIKit
 
 ///
 class ViewController: UIViewController {
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        self.view.backgroundColor = .yellow
+        NotificationCenter.default.addObserver(self,
+                                                  selector: #selector(applicationWillEnterForeground),
+                                                  name: UIApplication.willEnterForegroundNotification,
+                                                  object: nil)
+    
+        self.view.addSubview(labelName)
+        self.view.addSubview(chooseGameLabel)
+        self.view.addSubview(sumNumberLabel)
+        self.view.addSubview(guessNumberLabel)
+        
+        self.view.addSubview(guessNumberButton)
+        self.view.addSubview(sumNumberButton)
+        self.view.addSubview(showNewVC)
+        sumNumberButton.addTarget(self, action: #selector(sumNumber), for: .allTouchEvents)
+        guessNumberButton.addTarget(self, action: #selector(guessNumber), for: .allTouchEvents)
+        showNewVC.addTarget(self, action: #selector(showVC), for: .allTouchEvents)
+    }
+    
+    func showAlertHello() {
+        let alert = UIAlertController(title: "Welcome", message: "В форме ниже введите свое ФИО",
+                                      preferredStyle: .alert)
+        alert.addTextField { textField in
+            textField.placeholder = "Введите ФИО"
+        }
+        let action = UIAlertAction(title: "Добавить", style: .default) { _ in
+            self.labelName.text = "Hello, " + (alert.textFields?.first?.text ?? "Guest") + "!"
+        }
+        alert.addAction(action)
+        self.present(alert, animated: true)
+    }
+    var labelName: UILabel = {
+        let label = UILabel(frame: CGRect(x: 20, y: 80, width: 350, height: 60))
+        label.numberOfLines = 0
+        label.text = "Hello, !"
+        label.font = .boldSystemFont(ofSize: 24)
+        label.textColor = .black
+        label.textAlignment = .center
+        return label
+    }()
+    var chooseGameLabel: UILabel = {
```

Comment:
```suggestion
    var chooseGameLabel: UILabel = {
```
вью в самый верх посмотри в важном